### PR TITLE
Update ui.ts

### PR DESCRIPTION
**Pull request by falvarador**
#### What kind of changes does this PR include?
- New or updated content
- Translated content

#### Description
Add aside component default labels

### CODE REVIEW

File: src/i18n/es/ui.ts [hunk 1]
```diff
@@ -3,6 +3,11 @@ import { UIDictionary } from '../translation-checkers';
 export default UIDictionary({
 	'a11y.skipLink': 'Saltearse al contenido',
 	'navbar.a11yTitle': 'Navigación superior',
+    // Aside component default labels
```
Comment (agustinmulet) [inline src/i18n/es/ui.ts:6]: Nit:
```suggestion
  // Aside component default labels
```

**Review by agustinmulet — COMMENTED** [review]
Just a nitpick, nothing really necessary 😛

**Comment by kevinzunigacuellar** [comment]
LGTM! thank you @falvarador

**Comment by Yan-Thomas** [comment]
wow, a light speed review! Thanks so much everyone! @falvarador, I would love to see you in our Discord chat https://astro.build/chat 😄

**Pull request merged**